resolwe/flow/managers/workload_connectors/kubernetes.py

```diff
@@ -537,6 +537,34 @@ def _load_kubernetes_config(self):
             else kubernetes.config.load_kube_config()
         )
 
+    def optimize_job_scheduling(self, data: Data, job_description: dict):
+        """Optimize the scheduling by modifying the job description.
+
+        Moodify the job description to be a better fit on the scheduler. Currently this
+        is only used to schedule interactive/non-interactive processes.
```

Comment:
Scheduling for interactive/non-interactive processes is our use-case, but the idea behind it can change without changing the method and the doc string will become outdated. Use "schedule based on the scheduling class of the process" instead.

Reply:
Done.